api/test/test_helper/fake_android_plugin/fake_android_plugin.gd

```diff
@@ -40,16 +43,25 @@ var _asset_pack_location_store : Dictionary
 
 var _asset_pack_states_store : Dictionary
 
-var _fake_get_pack_states_info : FakeGetPackStatesInfo
-var _fake_cellular_confirmation_info : FakeCellularConfirmationInfo
-var _fake_remove_pack_info : FakeRemovePackInfo
+var _fetch_info : FakePackStatesInfo
+var _get_pack_states_info : FakePackStatesInfo
+var _cellular_confirmation_info : FakeCellularConfirmationInfo
+var _remove_pack_info : FakeRemovePackInfo
+
+var _asset_pack_state_updated_thread_pool : Array
```

Comment:
Might want to change this to `_asset_pack_state_updated_threads`. Thread pool implies a more [specific concept](https://en.wikipedia.org/wiki/Thread_pool).

Reply:
Thanks for mentioning this!